### PR TITLE
central added to settings menu

### PR DESCRIPTION
**Pull request by chandnirajasekaran**

### CODE REVIEW

File: QRealTime_dialog.py [hunk 1]
```diff
@@ -624,6 +620,82 @@ def getTable(self,XFormKey,importData,topElement,version= 'null'):
...
+        user=self.getValue(self.tr("user"))
+        password=self.getValue(self.tr("password"))
+        c_url=self.getValue('url')
+        # todo: add check to see if url exists
+        para={'format':'xml'}
+        data = {'email': user, 'password' : password}
+        headers = {"Content-Type": "application/json"}
+        requests.packages.urllib3.disable_warnings()
+        try:
+            response = requests.get(c_url+'v1/projects/4/forms/geopoint-form.xml', headers ={"Authorization": "Bearer " + usertoken})
```
Comment (shivareddyiirs) [inline QRealTime_dialog.py]: why url is appended with v1/projects/4........... seems to be a hard coding.
It should read from ui

**Comment by shivareddyiirs** [comment]
#55 
Also try to solve this issue as it would affect all servers

File: QRealTime_dialog.py [hunk 2]
```diff
@@ -624,6 +620,82 @@ def getTable(self,XFormKey,importData,topElement,version= 'null'):
         except Exception as e:
             print ('not able to fetch',e)
             return response,table
+
+class Central (Aggregate):
```
Comment (shivareddyiirs) [inline QRealTime_dialog.py]: I believe you should inherit from kobo CLASS as it has more similarity to ODK Central

**Review by shivareddyiirs — COMMENTED** [review]
Once changes are done.
Please test and post screenshots pre vs post changes.

good work!

**Comment by chandnirajasekaran** [comment]
Pre: 
Screen Shot 2021-02-17 at 7 13 38 PM

Post:
Screen Shot 2021-02-17 at 7 10 43 PM
Allows user to also specify project name for Central

**Pull request merged**